src/lib/testRun.test.ts

```
@@ -9,28 +11,50 @@ import {
-import fs from "node:fs";
+import { MaximEvaluatorAPI } from "./apis/evaluator";
+import { Semaphore } from "./utils/semaphore";
 
 // Load configuration for test environment
-const config = JSON.parse(fs.readFileSync(`${process.cwd()}/libs/maxim-js/testRunTestConfig.json`, "utf-8"));
+const config = JSON.parse(fs.readFileSync(`${process.cwd()}/testRunTestConfig.json`, "utf-8"));
 const env = "prod";
 
 // Environment variables validation
-let apiKey: string, workspaceId: string, workflowId: string, datasetId: string, promptVersionId: string;
+let baseUrl: string, apiKey: string, workspaceId: string, workflowId: string, datasetId: string, promptVersionId: string;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)